components/core/src/clp_s/ArchiveWriter.hpp

```diff
@@ -88,7 +88,10 @@ class ArchiveWriter {
     int m_compression_level{};
 
     std::shared_ptr<SchemaTree> m_schema_tree;
-    std::map<int32_t, SchemaWriter*> m_schema_id_to_writer;
+    std::map<int32_t, SchemaWriter*> m_id_to_schema_writer;
+
+    FileWriter m_table_file_writer, m_metadata_file_writer;
```

Comment:
```suggestion
    FileWriter m_table_file_writer;
    FileWriter m_metadata_file_writer;
```
I think our style guide recommends one declaration per line.